src/ui/src/components/forms/DatasourceForm/index.tsx

```diff
@@ -301,6 +353,46 @@ const DatasourceForm = () => {
           </EuiFormRow>
         </div>
       )}
+      {datasourceType === DataSourceTypes.BIGQUERY && (
+        <div>
+          <EuiPageHeader
+            iconType="https://cdn.worldvectorlogo.com/logos/google-bigquery-logo-1.svg"
```

Comment:
You added this file, should link to that

Reply:
good catch, replaced it in one spot, not the other